docs/data/toolpad/core/components/account/account.md

```diff
@@ -12,18 +12,20 @@ The `Account` component is a quick and easy way to display an account management
 
 ## States
 
-When signed out, the component renders as an inline sign in button within the dashboard layout. If a `session` object is present, the component is rendered as a dropdown containing the user's account details as well as an option to sign out.
+If a `session` object is present, the component is rendered as a dropdown containing the user's account details as well as an option to sign out. When signed out, the component renders as an inline sign in button within the dashboard layout.
 
-{{"demo": "AccountDemo.js", "bg": "gradient" }}
+{{"demo": "AccountDemo.js", "bg": "outlined" }}
 
 ## Customization
 
-### Components
+### Slot Props
 
 `Account` can take different labels for the sign in and sign out buttons through the `signInLabel` and `signOutLabel` props. Deeper changes can be made by passing in `slotProps` to the underlying components.
 
-{{"demo": "AccountCustom.js", "bg": "gradient" }}
```

Comment:
Pure visually, I wonder if it makes sense to show those demos inside of an empty `AppBar`.